plugins/circleci/readme.md

```diff
@@ -23,12 +25,13 @@ And install this plugin's hooks:
 npx dotcom-tool-kit --install
 ```
 
+// JJ: I don't understand what the below means?
 This plugin cannot currently automatically install itself to CircleCI configuration, so it will exit, and explain what you need to include in the config.
```

Comment:
**thought:** I also have no idea what it means. Do you @ivomurrell? If not, shall we remove it?